### ISSUE TITLE
Company registration process

### ISSUE THREAD

**Issue by joaopapereira** [AgileVentures/MetPlus_tracker#134]
As a developer
When an Employer registers
I need to send a confirmation email

**Comment by joaopapereira** [comment]
Blocked by #132

**Comment by joaopapereira** [comment]
In this story we need to ensure that after the creation of the employer an email is sent to the provided email address to confirm that the account was created and is waiting for approval

**Comment by patmbolger** [comment]
- [ ] Disable sending of Devise account "confirmation" email
- [ ] Send email to company contact stating that registration request has been received
- [ ] Create a Company record with status = 'Pending Review'
- [ ] Create a CompanyPerson record with status = 'Pending'
- [ ] Create associated User record that does *not* allow logins
- [ ] Disable sending of Devise account "confirmation" email upon User creation

Optional: (may be done in another story)
- [ ] Add task to agency_admin "to-do" list for follow up on registration request

This story to be consistent with system behavior as described here: http://www.agileventures.org/projects/metplus/documents/system-behavior (section "Action: Company Registration").
